glib-macros/tests/properties.rs

```diff
@@ -388,6 +388,56 @@ fn props() {
     );
 }
 
+mod ext_trait {
+    use glib::subclass::object::DerivedObjectProperties;
+    use glib::ObjectExt;
+
+    use glib::subclass::{prelude::ObjectImpl, types::ObjectSubclass};
+    use glib_macros::Properties;
+    use std::cell::RefCell;
+
+    pub mod imp {
+        use super::*;
+
+        #[derive(Properties, Default)]
+        #[properties(wrapper_type = super::Author, ext_trait = AuthorExt)]
```

Comment:
I think that is fine, as long as the trait is named something like `AuthorPropertiesExt`, so you can also have a similar trait for auto generated signals in the future for example.

Reply:
And that could also be a super-trait of the actual `Ext` trait. Maybe we should make that clearer in the docs?

Reply:
Now the trait defaults to `MyTypePropertiesExt`. I didn't specify anything about a possible super-trait `MyTypeExt`, because we don't have any macro to build that trait yet. The user can implement it as they wish.